zanata-war/src/main/java/org/zanata/servlet/FileUploadServlet.java

```diff
@@ -1,193 +0,0 @@
-/*
- * Copyright 2012, Red Hat, Inc. and individual contributors
- * as indicated by the @author tags. See the copyright.txt file in the
- * distribution for a full listing of individual contributors.
- *
- * This is free software; you can redistribute it and/or modify it
- * under the terms of the GNU Lesser General Public License as
- * published by the Free Software Foundation; either version 2.1 of
- * the License, or (at your option) any later version.
- *
- * This software is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this software; if not, write to the Free
- * Software Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA
- * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
- */
-package org.zanata.servlet;
-
-import java.io.IOException;
-import java.util.Collections;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Set;
-
-import javax.servlet.ServletException;
-import javax.servlet.http.HttpServlet;
-import javax.servlet.http.HttpServletRequest;
-import javax.servlet.http.HttpServletResponse;
-
-import lombok.extern.slf4j.Slf4j;
-
-import org.apache.commons.fileupload.FileItem;
-import org.apache.commons.fileupload.FileItemFactory;
-import org.apache.commons.fileupload.disk.DiskFileItemFactory;
-import org.apache.commons.fileupload.servlet.ServletFileUpload;
-import org.jboss.seam.servlet.ContextualHttpServletRequest;
-import org.zanata.common.LocaleId;
-import org.zanata.common.MergeType;
-import org.zanata.exception.ZanataServiceException;
-import org.zanata.rest.StringSet;
-import org.zanata.rest.dto.extensions.ExtensionType;
-import org.zanata.rest.dto.resource.TranslationsResource;
-import org.zanata.service.TranslationFileService;
-import org.zanata.service.TranslationService;
-import org.zanata.service.impl.TranslationFileServiceImpl;
-import org.zanata.service.impl.TranslationServiceImpl;
-import org.zanata.util.ServiceLocator;
-import org.zanata.webtrans.client.ui.FileUploadDialog;
-
-/**
- * Used for translation file upload from GWT editor. For endpoint, see servlet
- * binding for this class in web.xml
- *
- * @see {@link FileUploadDialog}
- *
- * @author Alex Eng <a href="mailto:aeng@redhat.com">aeng@redhat.com</a>
- *
- */
-@Slf4j
-public class FileUploadServlet extends HttpServlet {
```

Comment:
Removed upload support from GWT as it doesn't sync with the options in JSF. And its best to have 1 single place for uploading file.